client/performance-profiler/components/dashboard-content/index.tsx

```diff
@@ -17,7 +17,14 @@ export const PerformanceProfilerDashboardContent = ( {
 		<div className="performance-profiler-content">
 			<div className="l-block-wrapper container">
 				<PerformanceScore value={ overall_score * 100 } />
-				<CoreWebVitalsDisplay fcp={ fcp } lcp={ lcp } cls={ cls } inp={ inp } ttfb={ ttfb } />
+				<CoreWebVitalsDisplay
+					fcp={ fcp }
+					lcp={ lcp }
+					cls={ cls }
+					inp={ inp }
+					ttfb={ ttfb }
+					performanceReport={ performanceReport }
```

Comment:
Here we are passing only the needed data, so I think we might just pass `history` instead of the entire `performanceReport`

Reply:
Good point, done in 18e337c422cca5f7f2599d6248a117a7025f851c